kubernetes/devnet-9/dora/values.yaml

```diff
@@ -1,33 +1,28 @@
-# Ansible managed. Don't change this file manually. Template info: /vendor/collections/ansible_collections/ethpandaops/general/roles/generate_kubernetes_config/templates/light-beaconchain-explorer.yaml.j2
+# Ansible managed. Don't change this file manually. Template info: /vendor/collections/ansible_collections/ethpandaops/general/roles/generate_kubernetes_config/templates/dora.yaml.j2
 
-light-beaconchain-explorer:
-  fullnameOverride: light-beaconchain-explorer
+dora:
+  fullnameOverride: dora
   image:
-    repository: ethpandaops/dora-the-explorer
-    tag: master
+    repository: pk910/light-beaconchain-explorer
```

Comment:
this does not seem correct.

Reply:
fixed :)
(related to https://github.com/ethpandaops/ansible-collection-general/pull/140)